app/services/content_item_publisher.rb

```diff
@@ -125,6 +125,7 @@ def publish_transaction_via_publishing_api(base_path, publishing_app:, title:, c
     payload = {
       base_path: base_path,
       title: title,
+      update_type: 'major',
```

Comment:
It looks like the single quotes here are inconsistent with the rest of this file.